frontend/src/renderer/components/ImageImporter.svelte

```diff
@@ -0,0 +1,161 @@
+        files.accepted = [];
+        if (e.detail == null){
+            /* handles files being added from the select files button */
+            files.accepted = [...files.accepted, ...e.target.files];
+        }
+        else {
+            /* handles files being added from dragging and dropping */
+            files.accepted = [...e.detail?.acceptedFiles];
+        }
+
+        forEach(files.accepted, (f) => {
+            if (!find($processState.imageFilePaths, {id: f.path, name: f.name})) {
+                $processState.imageFilePaths.push({
+                    id: f.path,
+                    name: f.path
+                });
+            }
+        });
+        forEach($processState.imageFilePaths, function (f){
+           filePaths.push(f.name);
+        });
+        getThumbnails();
+        console.log($processState.imageFilePaths)
+    }
+
+    const remove = (item) => {
+        $processState.imageFilePaths = $processState.imageFilePaths.filter((value) => value.id !== item.id);
+    };
+
+    const removeAll = () => {
+        $processState.imageFilePaths = [];
+    };
+
+    function openAttachment() {
+        document.getElementById('attachment').click();
+    }
+
+
+</script>
+
+<main>
+    <br>
+    <Dropzone
+            on:drop={handleFilesSelect}
+            noClick
+            containerStyles="flex: 1;
+                            display: flex;
+                            flex-direction: column;
+                            align-items: center;
+                            padding: 20px;
+                            border-width: 2px;
+                            border-radius: 2px;
+                            border-color: #eeeeee;
+                            border-style: dashed;
+                            background-color: #1D1C1E;
+                            color: #bdbdbd;
+                            outline: none;
+                            transition: border 0.24s ease-in-out;
+                            background-color: #1D1C1E;
+                            border-radius: 10px;
+                            font-size: 30px;
+                            text-align: center;"
+            disableDefaultStyles
+            containerClasses="custom-dropzone">
+        <input type="file" class="file" id="attachment" style="display: none;" on:change={handleFilesSelect} multiple/>
+        <button
+                type="button"
+                class="file"
+                class:largeText
+                id="btnAttachment"
+                on:click={openAttachment}
+        >{label}
+            <div class="icon">
+                <svelte:component this={icon} size="1.5x" />
+            </div>
+        </button>
+
+        <br>
+        Drag and Drop Files Here
+        <br>
+        <br>
+        <article>
+            <ul>
+                {#if $processState.imageFilePaths?.length > 0}
+                    <button on:click={removeAll} class:largeText> Remove All</button>
```

Comment:
Just making a note here to move this to the top right corner. Do you think you could change it from "remove all" to just an x or something?

Reply:
I fell like just having an "x" will not tell the user what is happening.